kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -1,21 +1,415 @@
+                    >
+                      {{ $tr('shortNote') }}
+                    </button>
+                  </KGridItem>
+
+                  <KGridItem
+                    :layout12="{ span: 4 }"
+                  >
+                    <KIconButton
+                      class="float-item-left-style"
+                      icon="edit"
+                    />
+                  </KGridItem>
+                </KGrid>
+
+
+                <p
+                  class="choose-answer-style"
+                >
+                  {{ $tr('chooseAnswer') }}
+                </p>
+
+                <div
+                  v-for="(option,id) in placeholderOptions"
+                  :key="id"
+                >
+                  <AccordionQuizAnswer
+                    :optionValue="option.answer"
+                    :optionIndex="option.index"
+                    :isSelected="option.selected"
+                  />
+                </div>
+              </div>
+            </template>
+          </AccordionItem>
+        </div>
+      </template>
+    </AccordionContainer>
+
+    <hr
+      class="horizontal-border"
+      style="margin-top:100px;width:100%;"
+    >
+    <div class="bottom-bar-style">
+      <KButton
+        text="replace"
+        :primary="true"
+        class="float-button-right"
+      />
+    </div>
+
   </div>
 
 </template>
 
 
 <script>
 
+  import AccordionContainer from './AccordionContainer.vue';
+  import AccordionItem from './AccordionItem.vue';
+  import AccordionQuizAnswer from './AccordionQuizAnswer.vue';
+
   export default {
     name: 'ReplaceQuestions',
+    components: {
+      AccordionContainer,
+      AccordionItem,
+      AccordionQuizAnswer,
+    },
+    data() {
+      return {
+        placeholderOptions: [
+          {
+            index: 'A',
+            answer: 'bit',
+            selected: false,
+          },
+          {
+            index: 'B',
+            answer: 'bat',
+            selected: false,
+          },
+          {
+            index: 'C',
+            answer: 'but',
+            selected: false,
+          },
+          {
+            index: 'D',
+            answer: 'bite',
+            selected: false,
+          },
+          {
+            index: 'E',
+            answer: 'bet',
+            selected: false,
+          },
+          {
+            index: 'F',
+            answer: 'bait',
+            selected: true,
+          },
+        ],
+        placeholderList: [
+          'Letters and Sounds Practice',
+          'Mathematics and Geometry',
+          'Algebra Practice',
+          'Long and Short Vowel Patterns: VCV and VCC Practice',
+          'Letters and Sounds Practice',
+          'Mathematics and Geometry',
+          'Algebra Practice',
+          'Long and Short Vowel Patterns: VCV and VCC Practice',
+        ],
+      };
+    },
+    $trs: {
+      replaceQuestionText: {
+        message: 'Replace questions',
```

Comment:
I thought we will be using the common translations approach for all new features, is there consensus about it? @marcellamaki 

It's described here in more detail https://kolibri-dev.readthedocs.io/en/develop/i18n.html#common-string-modules